server/package.json

```diff
@@ -59,6 +59,7 @@
     "express": "^4.18.1",
     "fs-extra": "^10.1.0",
     "lodash": "^4.17.21",
+    "node-fetch": "2",
```

Comment:
Install `node-fetch` so that we can send requests in server.
I tried to use `@nestjs/axios` at first, but it seems that we intended to do some kind of singleton stuff, so inject services in constructor would not work.